chunk_vcf.py

```diff
@@ -135,9 +143,11 @@ def prepare_and_split_vcf(vcf, outfolder, size, logpath=None):
               help='Path to a folder where VCF will be written if the input VCF is larger than the size argument')
 @click.option('-s', '--size', required=True, type=int,
               help='Size in bytes. If the VCF exceed this size, it will be split into 2, 3 or 4 VCFs')
+@click.option('-ref', '--reference', required=True,
```

Comment:
See previous comment about `type=click.Choice([])`

Reply:
solved in 58a48a3